site/gatsby-site/playwright/e2e/integration/cset.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { expect, Page } from '@playwright/test';
+import { gql } from '@apollo/client';
+import { query, test } from '../../utils';
+
+const urls = [
+  { namespace: 'CSETv0', url: '/taxonomy/csetv0' },
+  { namespace: 'CSETv1', url: '/taxonomy/csetv1' },
+];
+
+urls.forEach(({ namespace, url }) => {
+  test(`successfully loads ${namespace}`, async ({ page }) => {
+    await page.goto(url);
+  });
+
+  if (namespace === 'CSETv0') {
```

Comment:
Can we remove this condition? Why we don't test the `CSETv1`?

Reply:
Because the CSETv1 page is completely different, so it can't be tested in the same way. Maybe it needs a different test to be added later